magento/pwa-studio
# PWA-1555: cypress test to verify row

## Description

To verify that the Page Builder Row content type renders correctly on Venia for each content type property.

## Related Issue







Closes [PWA-1555](https://jira.corp.magento.com/browse/PWA-1555): Verify Row

## Acceptance





### Verification Stakeholders



### Specification




## Verification Steps

- Go to venia-integration-test dir
- Run yarn install
- Run yarn test
- Run row.spec.js from cypress ui
- Cypress row.spec.js should pass

#### Test scenario(s) for direct fix/feature





#### Test scenario(s) for any existing impacted features/areas





#### Test scenario(s) for any Magento Backend Supported Configurations





#### Is Browser/Device testing needed?





#### Any ad-hoc/edge case scenarios that need to be considered?




## Screenshots / Screen Captures (if appropriate)

## Breaking Changes (if any)





## Checklist



-   I have added tests to cover my changes, if necessary.
-   I have added translations for new strings, if necessary.
-   I have updated the documentation accordingly, if necessary.
